### PR TITLE
Documentation for LandXML

### PR DESCRIPTION
**Pull request by psolyca**
I will update this PR and marge master after #143.

### CODE REVIEW

**Comment by psolyca** [comment]
@steko good to review ^^

File: docs/output_formats/of_landxml.rst [hunk 1]
```diff
@@ -0,0 +1,178 @@
+.. _LandXML website: http://www.landxml.org/
+
+.. _if_landxml:
```
Comment (steko) [inline docs/output_formats/of_landxml.rst]: This should be `_of_landxml`

File: docs/output_formats/of_landxml.rst [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+=========================
+:mod:`tops_landxml` -- LandXML
+=========================
```
Comment (steko) [inline docs/output_formats/of_landxml.rst]: Sphinx complains with WARNING: Title overline too short.

File: docs/input_formats/if_landxml.rst [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+The official documentation about the format is provided on the `LandXML website`_.
+
+LandXML is a specialized XML (eXtensible Mark-up Language) data file format
+containing civil engineering and survey measurement data commonly used in the
+Land Development and Transportation Industries.
+
+For example::
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <LandXML xmlns="http://www.landxml.org/schema/LandXML-1.1" time="15:49:09" date="2006-06-19" version="0" language="English" readOnly="false">
```
Comment (steko) [inline docs/input_formats/if_landxml.rst]: the `xmlns` attribute should point to LandXML-1.2 as in the output format example

File: docs/input_formats/if_landxml.rst [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
++------------------+--------------------+------------------+--------------------------+
+|                  | setupID            |                  | station_id               |
++------------------+--------------------+------------------+--------------------------+
+| Backsight*       |                    | ObservationGroup |                          |
++------------------+--------------------+------------------+--------------------------+
+|                  | circle             |                  | circle                   |
++------------------+--------------------+------------------+--------------------------+
+|                  | setupID            |                  | "setup" + id             |
++------------------+--------------------+------------------+--------------------------+
+| BacksightPoint*  |                    | ObservationGroup |                          |
```
Comment (steko) [inline docs/input_formats/if_landxml.rst]: I thought you implemented Backsight in the other PR ?

**Review by steko — CHANGES_REQUESTED** [review]
A few minor tweaks

File: docs/input_formats/if_landxml.rst (hunk 4 above)
Comment (psolyca) [reply docs/input_formats/if_landxml.rst]: Hahahah yes, I write documentation before the change ^^
Backsight was already here but not on the right place.

File: docs/input_formats/if_landxml.rst (hunk 3 above)
Comment (psolyca) [reply docs/input_formats/if_landxml.rst]: Forgot this one ^^

**Review by steko — APPROVED** [review]

**Pull request merged**